src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -33,19 +29,16 @@ public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
         Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
+        List<StationResponse> stations = StationResponse.of(persistLine.findStations());
+
         return LineResponse.of(persistLine, stations);
```

Comment:
```suggestion
        return LineResponse.of(persistLine);
```

와 같이 작성해볼 수 있을 것 같아요 :)